README.md

```diff
@@ -1,14 +1,20 @@
 # Project
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This utility can be used to detect the supported TLS version for a given CosmosDB account.
 
-As the maintainer of this project, please make a few updates:
+Note that this utility would only be able to test the account against TLS versions supported by the client system running the script.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Supported TLS version:
+- TLS V1
+- TLS V1.1
+- TLS V1.2
+- TLS V1.3
+
+Run the utility with a given CosmosDB endpoint as follows
```

Comment:
should tell folks which version of Python they should use and how to install the packages in requirements.txt

Reply:
Our Python SDK for Cosmos is a good example of a lean readme: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos

Reply:
Added to readme